src/lib/tabs/tab-group.scss

```diff
@@ -20,6 +20,7 @@
 @media ($mat-xsmall) {
   .mat-tab-label {
     min-width: 72px;
+    padding: 0 12px;
```

Comment:
Is it only `xsmall` that should have the denser look?

Reply:
Bumped to small per our convo.